index.js

```diff
@@ -56,6 +56,25 @@ module.exports = {
     'import/no-named-default': 'error',
     'import/no-webpack-loader-syntax': 'error',
     'import/prefer-default-export': 'error',
+    'jsdoc/check-examples': 0,
+    'jsdoc/check-param-names': 1,
+    'jsdoc/check-tag-names': 1,
+    'jsdoc/check-types': 1,
+    'jsdoc/newline-after-description': 0,
```

Comment:
Why `off` for `jsdoc/newline-after-description`. Is it problematic?

Reply:
Without counting proper statistics, it would seem like in the backend the style is not to use newline. Turning this on would create a large amount of new issues.

Reply:
So we more often do:

```
/**
 * Some desc
 * @param paramName {type} Param desc
```

than:

```
/**
 * Some desc
 *
 * @param paramName {type} Param desc
```

?

I think latter is leaner

Reply:
+1 from me for the second example

Reply:
If I turn this on there will be 122 additional errors.

Reply:
I would turn this on later if we want to.

Reply:
I think count of how many errors it invokes for whole code base should not be used as an argument (we're not obliged to clear them upfront).

We will need to fix it only for existing files, and it's very easy to fix (adding newline).

So I would focus purely on which style we prefer.  We have 2 after second version, what's your preference @mfonsen ?